### PR TITLE
Emission / LFO split

### PR DESCRIPTION
**Pull request by dleard**
emisison and lfo have split properly (as far as my tests can tell).
The new table additonal_reportable_activity has been created, but it appears to not be receiving any data...I'm not sure why.
Run make unit and there is a failing test at
isnt_empty(select * from ggircs.additional_reportable_activity)

### CODE REVIEW

**Comment by dleard** [comment]
Branched off #62

File: deploy/function_export_mv_to_table.sql [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+
+      -- Create FK/PK relation between Fuel and Unit
+      alter table ggircs.fuel add column unit_id int;
+      create index ggircs_fuel_unit_index on ggircs.fuel (unit_id);
+      update ggircs.fuel set unit_id = unit.id from ggircs.unit
+          where fuel.ghgr_import_id = unit.ghgr_import_id and fuel.process_idx = unit.process_idx  and fuel.sub_process_idx = unit.sub_process_idx
+            and fuel.activity_name = unit.activity_name and fuel.units_idx = unit.units_idx and fuel.unit_idx = unit.unit_idx;
+      alter table ggircs.fuel add constraint ggircs_fuel_unit_foreign_key foreign key (unit_id) references ggircs.unit(id);
+
+    -- Create FK/PK relation between Organisation and Parent Organisation
```
Comment (dleard) [inline deploy/function_export_mv_to_table.sql]: Is this relation backwards?

File: deploy/function_export_mv_to_table.sql [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+            and unit.activity_name = activity.activity_name;
+      alter table ggircs.unit add constraint ggircs_unit_activity_foreign_key foreign key (activity_id) references ggircs.activity(id);
+
+  end;
+
+$$ language plpgsql volatile ;
+
+commit;
+
+
```
Comment (dleard) [inline deploy/function_export_mv_to_table.sql]: On deploy, should this function be called? ie: right here?

File: deploy/function_export_mv_to_table.sql (hunk 1 above)
Comment (hamzajaved) [reply deploy/function_export_mv_to_table.sql]: it's a many to many relationship. So we will need FKs on both...

**Pull request merged**